obspy/core/stream.py

```diff
@@ -1592,12 +1606,92 @@ def slice(self, starttime=None, endtime=None, keep_empty_traces=False):
         tmp.traces = []
         new = tmp.copy()
         for trace in self:
-            sliced_trace = trace.slice(starttime=starttime, endtime=endtime)
+            sliced_trace = trace.slice(starttime=starttime, endtime=endtime,
+                                       nearest_sample=nearest_sample)
             if keep_empty_traces is False and not sliced_trace.stats.npts:
                 continue
             new.append(sliced_trace)
         return new
 
+    def slide(self, window_length, step, offset=0,
+              include_partial_windows=False, nearest_sample=True):
+        """
+        Generator yielding equal length sliding windows of the Stream.
+
+        Please keep in mind that it only returns a new view of the original
+        data. Any modifications are applied to the original data as well. If
```

Comment:
I know it's implied, but it might be good to explicitly state that it would also affect all following windows.